src/content/pt/2019/resource-hints.md

```diff
@@ -228,15 +228,15 @@ Embora outros elementos HTML suportem o atributo crossorigin, esta análise apen
 )
 }}
 
-88% das instâncias de dicas de recursos usam o atributo `as`. Quando `as` especificado, é predominantemente usado para scripts: 92% do uso é para scripts, 3% para fontes e 3% para estilos. Isso não é surpreendente, dado o papel proeminente que os scripts desempenham na arquitetura da maioria dos sites, bem como a alta frequência com que os scripts são usados ​​como vetores de ataque (tornando, portanto, particularmente importante que os scripts obtenham o CSP correto aplicado a eles).
+88% das instâncias de dicas de recursos usam o atributo `as`. Quando `as` é especificado, é predominantemente usado para scripts: 92% do uso é para scripts, 3% para fontes e 3% para estilos. Isso não é surpreendente, dado o papel proeminente que os scripts desempenham na arquitetura da maioria dos sites, bem como a alta frequência com que os scripts são usados ​​como vetores de ataque (tornando, portanto, particularmente importante que os scripts obtenham o CSP correto aplicado a eles).
 
 ## O futuro
 
 No momento, não há propostas para expandir o conjunto atual de dicas de recursos. No entanto, dicas de prioridade e carregamento lento nativo são duas tecnologias propostas que são semelhantes em espírito às dicas de recursos, pois fornecem APIs para otimizar o processo de carregamento.
 
-### Dicas de prioridade
+### Sugestões prioritárias
```

Comment:
I think here the best translation is "Dicas de prioridade". What do you think?

Reply:
> I think here the best translation is "Dicas de prioridade". What do you think?

@emanuelgsouza, would also be very nice, we can substitute if you want.

Reply:
Great, so if you can, please change it. Thanks